includes/class-boldgrid-inspirations.php

```diff
@@ -318,6 +318,33 @@ private function queue_hooks() {
 				'filter_allowed_html',
 			), 10, 2
 		);
+
+		// Create Onboarding Tasks when deployment is complete.
```

Comment:
I mentioned this before, but it feels like we should be able to add tasks when need be... e.g. a month from now we may realize users really need to do xy&z, we need a way to add that task.

Reply:
I intentionally didn't what that to be a thing.... If we add new items, then it should be something that only gets added on a new install or redeployment. We don't want users to have gone to 100% completion, and then have that number keep changing on them. That would get really really annoying